packages/vkui/src/components/CustomScrollView/CustomScrollView.tsx

```diff
@@ -12,6 +12,7 @@ export interface CustomScrollViewProps extends DOMProps, TrackerOptionsProps {
   windowResize?: boolean;
   boxRef?: React.Ref<HTMLDivElement>;
   className?: HTMLDivElement['className'];
+  onScroll?: (event: React.UIEvent<HTMLDivElement>) => void;
```

Comment:
```suggestion
  onScroll?(event: React.UIEvent<HTMLDivElement>): void;
```
чтобы как честный метод был